.circleci/config.yml

```diff
@@ -370,8 +387,13 @@ jobs:
   ios-release-tag:
     <<: *base-job
     steps:
-      - ensure-tag-is-semver-like:
-          tag: ${CIRCLE_TAG}
+      - run:
+          name: Verify that tag $CIRCLE_TAG adheres to the Semantic Versioning format
+          command: |
+            if [[ ! "$CIRCLE_TAG" =~ ^v[0-9]+\.[0-9]+\.[0-9]+ ]]; then
+              echo "Skipping job"
+              exit 1
```

Comment:
Let's say we're failing the job and why instead of skipping it.

Reply:
Will update in a subsequent PR